packages/vega-parser/src/parsers/title.js

```diff
@@ -66,7 +66,7 @@ function groupEncode(_, userEncode) {
     limit:      _('limit'),
     frame:      _('frame'),
     offset:     _('offset') || 0,
-    padding:    _('subtitlePadding')
+    padding:    _('subtitlePadding'),
```

Comment:
Remove extra `,`

Reply:
Vega does not enforce trailing commas so I'll ignore this. Like here https://github.com/vega/vega/blob/master/packages/vega-parser/src/parsers/legend.js#L152